packages/websocket-test/CMakeLists.txt

```diff
@@ -49,3 +51,9 @@ foreach(resource
   )
 
 endforeach()
+
+# Windows dll dependencies installation
+if(WIN32)
+  find_file(JSONDLL NAMES "soss-json.dll" PATHS "${soss-json_DIR}" PATH_SUFFIXES "lib" )
+  install (FILES ${JSONDLL} DESTINATION "${CMAKE_INSTALL_PREFIX}/../bin")
```

Comment:
Similar to my comment before about copying the `soss-mock` dll, is this really necessary, or is it so that users don't need to run as many batch scripts before running the test?

Reply:
Is because is easier to copy all dll dependencies to one single folder and then add that foler to the path in order Windows to be able to load dependencies running soss or tests.